test/APIBasedAuth.test.ts

```diff
@@ -581,8 +581,8 @@ describe('with auth successfully created', () => {
 
     const input: Omit<APIBasedAuth.InitiateSignupData, 'clientId'> = {
       email: 'test.user@test.com',
-      phone: undefined,
-      username: 'test.user',
+      phone: '+12223334444',
+      username: undefined,
```

Comment:
Should you have two tests? One where a username is provided and one where it is not? Or is that just splitting hairs?

Reply:
since it doesn't really change the internal code, I didn't think it was needed. But it would be a good example of the different ways it could be used. 

I'll add one